### PR TITLE
Kernel: Remove misleading VGA code, simplify graphics initialization somewhat

### PR DESCRIPTION
**Pull request by supercomputer7**

### CODE REVIEW

File: Kernel/Graphics/GraphicsManagement.cpp [hunk 1]
```diff
@@ -174,45 +174,18 @@ UNMAP_AFTER_INIT void GraphicsManagement::initialize_preset_resolution_generic_d
...
-     * In that situation, the kernel will assume that VGA text mode was already
-     * initialized, but will still try to modeset it. No switching to graphical
-     * environment is allowed in this case.
+    /* Explanation on the flow here:
      *
-     * By default, the kernel assumes that no framebuffer was created until it
-     * was proven that there's an existing framebuffer or we can modeset the
-     * screen resolution to create a framebuffer.
+     * If the user chose to disable graphics support entirely, then all we can do
+     * is to setup a plain old VGA text console and exit this function.
```
Comment (linusg) [inline Kernel/Graphics/GraphicsManagement.cpp:180]:
```suggestion
     * is to set up a plain old VGA text console and exit this function.
```

File: Kernel/Graphics/GraphicsManagement.cpp [hunk 2]
```diff
@@ -174,45 +174,18 @@ UNMAP_AFTER_INIT void GraphicsManagement::initialize_preset_resolution_generic_d
...
+     * Otherwise, we either try to find a device that we natively support it so
```
Comment (linusg) [inline Kernel/Graphics/GraphicsManagement.cpp:181]:
```suggestion
     * Otherwise, we either try to find a device that we natively support so
```

**Review by linusg — COMMENTED** [review]
typo in first commit: `remainments` -> `remains`? `remaining parts`?

**Comment by supercomputer7** [comment]
@linusg The comments of yours were addressed in the last rebase now :)

**Pull request merged**